cmd/compose/compose.go

```diff
@@ -142,13 +142,16 @@ func (o *projectOptions) WithServices(fn ProjectServicesFunc) func(cmd *cobra.Co
 }
 
 func (o *projectOptions) addProjectFlags(f *pflag.FlagSet) {
+	var compatibility bool
 	f.StringArrayVar(&o.Profiles, "profile", []string{}, "Specify a profile to enable")
 	f.StringVarP(&o.ProjectName, "project-name", "p", "", "Project name")
 	f.StringArrayVarP(&o.ConfigPaths, "file", "f", []string{}, "Compose configuration files")
 	f.StringVar(&o.EnvFile, "env-file", "", "Specify an alternate environment file.")
 	f.StringVar(&o.ProjectDir, "project-directory", "", "Specify an alternate working directory\n(default: the path of the Compose file)")
 	f.StringVar(&o.WorkDir, "workdir", "", "DEPRECATED! USE --project-directory INSTEAD.\nSpecify an alternate working directory\n(default: the path of the Compose file)")
+	f.BoolVar(&compatibility, "compatibility", false, "DEPRECATED")
```

Comment:
I think it could just be
```suggestion
	_ = f.Bool("compatibility", false, "DEPRECATED")
```